roles/scm/git/tasks/prep.yml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Prepare git credentials (SSH)
+  block:
+    - name: Define SSH folder location
+      set_fact:
+        ssh_dir: "{{ playbook_dir}}/.ssh"
```

Comment:
it's not a given that the playbook_dir is writable (various execution environments)  - please use a proper temp location instead (i.e.: tempdir controlled by the environment/host).

Reply:
updated to use ansible.builtin.tempfile